### PR TITLE
feat: add resizable inner sidebar

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

- inner sidebar now resizable for SQL editor and Table Editor

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

https://github.com/supabase/supabase/assets/8291514/1f3d04ef-df86-4398-b7b3-42a9effe950d

### CODE REVIEW

**Comment by MildTomato** [comment]
Needs some testing.

Also some unanswered questions in slack regarding where maybe we should disable it

File: packages/ui/index.tsx [hunk 1]
```diff
@@ -204,6 +204,12 @@ export {
 
 export { ScrollArea, ScrollBar } from './src/components/shadcn/ui/scroll-area'
 
+export {
+  ResizablePanelGroup as ResizablePanelGroup_Shadcn_,
```
Comment (ivasilov) [inline packages/ui/index.tsx:208]: No need for `_shadcn_` suffix since there's no conflict with another component.

**Review by ivasilov — APPROVED** [review]

File: packages/ui/index.tsx (hunk 1 above)
Comment (MildTomato) [reply packages/ui/index.tsx:208]: agreeeed

**Review by ivasilov — APPROVED** [review]

**Comment by MildTomato** [comment]
ok, merging

**Pull request merged**